src/csvcubed/models/inspect/sparqlresults.py

```diff
@@ -32,9 +32,11 @@ class CatalogMetadataResult:
     """
 
     dataset_uri: str
-    """Data set here doesn't necessarily mean the qb:DataSet. It means either the qb:DataSet or the skos:ConceptScheme."""
+    """`dataset` here doesn't necessarily mean the dcat:DataSet. It means either the dcat:DataSet or the skos:ConceptScheme (which is also a dcat:Dataset)."""
```

Comment:
"""`dataset` here doesn't necessarily mean the **qb:DataSet**. It means either the dcat:DataSet or the skos:ConceptScheme (which is also a dcat:Dataset)."""